cli/command/service/list_test.go

```diff
@@ -22,7 +29,315 @@ func TestServiceListOrder(t *testing.T) {
 		},
 	})
 	cmd := newListCommand(cli)
+	cmd.SetArgs([]string{})
```

Comment:
This was to allow running the test separately in my IDE; when pre-compiling the test and running it, this test would otherwise pick-up the `-test.run` flags as arguments, and fail the  test;

```bash
go test -c -o /tmp/foo.test github.com/docker/cli/cli/command/service \
go tool test2json -t /tmp/foo.test -test.v -test.run '^TestServiceListOrder$'

...
list_test.go:34: assertion failed: error is not nil: \"ls\" accepts no arguments.\n"}
```